packages/@dcl/inspector/src/components/Renderer/Metrics/Metrics.tsx

```diff
@@ -0,0 +1,112 @@
+import { IoGridOutline as SquaresGridIcon, IoAlertCircleOutline as AlertIcon } from 'react-icons/io5'
+
+import { withSdk, WithSdkProps } from '../../../hoc/withSdk'
+import { useChange } from '../../../hooks/sdk/useChange'
+import { useOutsideClick } from '../../../hooks/useOutsideClick'
+import { Button } from '../../Button'
+import { getSceneLimits } from './utils'
+import type { Metrics } from './types'
+
+import './Metrics.css'
+
+const ICON_SIZE = 18
+
+const Metrics = withSdk<WithSdkProps>(({ sdk }) => {
+  const [showMetrics, setShowMetrics] = React.useState(false)
+  const [metrics, setMetrics] = React.useState<Partial<Metrics>>({
+    triangles: 0,
+    entities: 0,
+    bodies: 0,
+    materials: 0,
+    textures: 0
+  })
+
+  const handleUpdateMetrics = useCallback(() => {
+    const ROOT_ENGINE = sdk.engine.RootEntity
+    const triangles = sdk.scene.meshes.reduce((acc, mesh) => acc + mesh.getTotalVertices(), 0)
+    const entities = (sdk.components.Nodes.getOrNull(ROOT_ENGINE)?.value ?? [ROOT_ENGINE]).length - 1
+    setMetrics({
+      triangles: triangles,
+      entities: entities,
+      bodies: sdk.scene.meshes.length,
+      materials: sdk.scene.materials.length,
+      textures: sdk.scene.textures.length
+    })
+  }, [sdk])
+
+  useEffect(handleUpdateMetrics, [handleUpdateMetrics])
+
+  useChange(handleUpdateMetrics, [handleUpdateMetrics])
+
+  const scene = useMemo(() => {
+    return sdk.components.Scene.get(sdk.engine.RootEntity)
+  }, [sdk])
+
+  const limits = useMemo<Metrics>(() => {
+    const parcels = scene.layout.base.x * scene.layout.base.y
+    return getSceneLimits(parcels)
+  }, [scene])
+
+  const limitsExceeded = useMemo<Record<string, boolean>>(() => {
+    return Object.fromEntries(
+      Object.entries(metrics)
+        .map(([key, value]) => [key, value > limits[key as keyof Metrics]])
+        .filter(([, value]) => value)
+    )
+  }, [metrics, limits])
+
+  const handleToggleMetricsOverlay = useCallback(
+    (e: React.MouseEvent<HTMLButtonElement> | MouseEvent) => {
+      e.preventDefault()
+      e.stopPropagation()
+      setShowMetrics((value) => !value)
+    },
+    [showMetrics, setShowMetrics]
+  )
+
+  const overlayRef = useOutsideClick(handleToggleMetricsOverlay)
+
+  return (
+    <div className="Metrics">
+      <div className="Buttons">
+        <Button
+          className={cx({ Active: showMetrics, LimitExceeded: Object.values(limitsExceeded).length > 0 })}
+          onClick={handleToggleMetricsOverlay}
+        >
+          <SquaresGridIcon size={ICON_SIZE} />
+        </Button>
+      </div>
+      {Object.values(limitsExceeded).length > 0 && (
+        <div className="LimitExceeded">
+          <AlertIcon />
+          Too many {Object.keys(limitsExceeded)[0].toUpperCase()}
+        </div>
+      )}
+      {showMetrics && (
+        <div ref={overlayRef} className="Overlay">
+          <h2 className="Header">
+            {scene.layout.base.x}x{scene.layout.base.y} LAND
```

Comment:
the "COLSxROWS" layout only makes sense for the builder, but for scenes with irregular shapes it does not work. For example you could have a scene with the shape of an "S" and that is not translatable to a COLSxROWS format.

Also `layout.base` is not being used correctly in this context. The base is the corner of the scene where to originate the x,y,z coordinate system, but is not the number of columns or rows neither.

Also for Worlds the LAND part is not relevant.

So I would say we simply remove the COLSxROWS label from the UI.